src/components/pick-list/shared-list-logic.ts

```diff
@@ -11,6 +11,11 @@ type List<T> = T extends PickList ? PickList : ValueList;
 
 export type ListFocusId = "filter";
 
+export interface CalciteListFilterDetail {
+  calciteListFilter: ItemData;
```

Comment:
Can we introduce a `filterText` (or similar) prop on the lists to skip the event payload?

Reply:
@jcfranco what about the `ItemData`? Right now we just have `@State() dataForFilter: ItemData = [];`?

I like the `filterText` idea because that allows the users to set the filter as well.

Reply:
Would we want `filterData` prop as well? readonly.

Reply:
Updated! Let me know how it looks now @jcfranco